sdk/python/cli.py

```diff
@@ -0,0 +1,285 @@
+import os
+import shutil
+from io import BytesIO
+from zipfile import ZipFile
+import click
+import requests
+import json
+import subprocess
+from pathlib import Path
+from dotenv import load_dotenv
+from typing import Tuple
+
+
+@click.group()
+def main():
+    """Sentience CLI"""
+    pass
+
+
+@main.group()
+def agent():
+    """Agent-related commands."""
+    pass
+
+
+@agent.command()
+def init() -> None:
+    """Create a new Agent folder template in the current directory."""
+    agent_name = click.prompt("Enter agent name", type=str)
+    docker_username = click.prompt("Enter Docker username", type=str)
```

Comment:
1) shouldn't credentials be in `.env` files instead of this prompt? 
2) AFAIK you can use api keys for Docker, instead of username/password https://docs.docker.com/security/for-developers/access-tokens/